README.md

```diff
@@ -1,2 +1,31 @@
 # semantic_html_refactor
+
 Refactoring a marketing agency website to use semantic HTML
+
+## Summary
+
+##### I have refactored a website for "Horiseon Inc" in this project. I have factored in the alt text to make the website more accessible for all users. I understand that search engine optimisation is a vital part of the company, so I have even decided to add as many semantic HTML elements as possible by removing the div elements in the previous code. To stop a higher likelihood of having bugs, I have reduced the repeated CSS in the CSS files and ensured that HTML changes are followed through in the CSS
+
+## About Me
+
+I am working currently as a Healthcare Support Worker Apprentice in Birmingham. I have always loved computers, and it has been a fascination of mine for life; I
+finally decided to explore my adoration for coding by starting. :nerd_face: :grin: I am passionate about teaching, and I hope that my GitHub repositories will show the expanse of all the knowledge that I can use to educate others.
+
+## Before
+
+### Horiseon webpage before ![ Horiseon webpage before](/assets/images/screenshots/beforewebsite.png "before")
+
+## After
+
+### Horiseon webpage after![ Horiseon webpage after](/assets/images/screenshots/afterwebsite.png "after")
+
```

Comment:
These seem to be the same image. Try to use images that show a difference between them.